core/src/ledger/parameters/storage.rs

```diff
@@ -183,3 +192,15 @@ pub fn get_faucet_account_key() -> Key {
 pub fn get_wrapper_tx_fees_key() -> Key {
     get_wrapper_tx_fees_key_at_addr(ADDRESS)
 }
+
+/// Storage key used for the max signatures per transaction key
+pub fn get_max_signatures_per_transaction_key() -> Key {
+    Key {
+        segments: vec![
+            DbKeySeg::AddressSeg(ADDRESS),
+            DbKeySeg::StringSeg(
+                Keys::VALUES.max_signatures_per_transaction.to_string(),
+            ),
+        ],
+    }
```

Comment:
```suggestion
    is_max_signatures_per_transaction_key_at_addr(key, &ADDRESS)
```
I think this should work. the `StorageKeys` macro generates these predicate functions automatically